packages/@okta/vuepress-site/docs/concepts/role-assignment/index.md

```diff
@@ -52,6 +64,17 @@ Therefore, when dealing with Custom Roles, these three entities always exist:
 * A principal: Either a Group or a User - known as a Member of the Binding
 * A Resource Set: Identified by its `id`
 
+### Custom Role Assignment for IAM-based Standard Roles
+<ApiLifecycle access="ea" />
+When using IAM-based Standard Roles the same concepts as Custom Roles apply with the following distinctions:
+1. IAM-based standard roles can only be used with pre-defined Resource Sets.
+2. For both IAM-based Standard Roles and Resource Sets, there is a predefined constant `id`. This `id` will always be the `type` of the Role or Resource Set.
```

Comment:
nit: `pre-defined` in 1. and `predefined` in 2. Apparently they are both in the dictionary but couldn't help myself from commenting.

Reply:
Ah good catch. @brianduffield-okta do you have any preference which to use?

Reply:
@alifatolahi-okta we don't have an internal style. In these cases we defer to the Microsoft Manual of style: https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/p/pre. So, in this case, I'd suggest `predefined`.

Reply:
@alifatolahi-okta FYI, I can make this minor update when merging the release branch into our main branch later this evening. No need to update here.